android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java

```diff
@@ -257,37 +275,7 @@ public static void onNewIntent(Intent intent) {
         }
     }
 
-    /**
-     * Activity lifecycle method which should be called from
-     * {@code Activity.onUserLeaveHint} so we can do the required internal
-     * processing.
-     *
-     * This is currently not mandatory.
-     */
-    public static void onUserLeaveHint() {
-        sendEvent("onUserLeaveHint", null);
-    }
-
-    /**
-     * Helper function to send an event to JavaScript.
-     *
-     * @param eventName {@code String} containing the event name.
-     * @param params {@code WritableMap} optional ancillary data for the event.
-     */
-    private static void sendEvent(
-            String eventName,
-            @Nullable WritableMap params) {
-        if (reactInstanceManager != null) {
-            ReactContext reactContext
-                = reactInstanceManager.getCurrentReactContext();
-            if (reactContext != null) {
-                reactContext
-                    .getJSModule(
-                        DeviceEventManagerModule.RCTDeviceEventEmitter.class)
-                    .emit(eventName, params);
-            }
-        }
-    }
+    private String conferenceUrl;
```

Comment:
does this not need to be synchronized?

Reply:
I don't know... currently only one thread writes there and one reads, so maybe volatile keyword would be enough